lib/service/service.go

```diff
@@ -2244,6 +2244,10 @@ func (process *TeleportProcess) initSSH() error {
 		// TODO: are we missing rm.Close()
 
 		// make sure the namespace exists
+		if ns := cfg.SSH.Namespace; ns != "" && ns != apidefaults.Namespace {
+			return trace.BadParameter("cannot start with custom namespace %q, custom namespaces are deprecated. "+
+				"use builtin namespace '%s', or omit the 'namespace' config option.", ns, apidefaults.Namespace)
```

Comment:
```suggestion
				"use builtin namespace %q, or omit the 'namespace' config option.", ns, apidefaults.Namespace)
```

I would use `%q` or `'%s'` consistently instead of mixing the two in the same error message